src/scalar.rs

```diff
@@ -31,6 +31,7 @@ use std::ops::{Index, IndexMut};
 
 use rand::Rng;
 
+// XXX should these be in a utility module ?
```

Comment:
We could move these to a `utils` module, but if we do that, should we also move `util` to something more appropriately named, like `subtle`?